### PR TITLE
Feature: creating order endpoints

### PR DESCRIPTION
**Pull request by eapg**
* Adding endpoints for order controller
* Adding e2e test for order controller endpoints

### CODE REVIEW

File: src/flask/routes/order.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    @order_blueprint.route("/orders/<order_id>", methods=["DELETE"])
+    def delete_order_by_id(order_id):
+        json_order_data = request.get_json()
+        order = order_schema.load(json_order_data)
+        order_controller.delete_by_id(order_id, order)
+        delete_response = make_response("", HttpStatus.OK.value)
+        return delete_response
+
+    @order_blueprint.route(
+        "/orders/by_order_status/<order_status>/<order_limit>", methods=["GET"]
```
Comment (reynoldmorel) [inline src/flask/routes/order.py:56]: Send the limit as a query param instead of path param

File: src/flask/routes/order.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    )
+    def get_orders_by_status(order_status, order_limit):
+        orders_by_status = order_controller.get_orders_by_status(
+            order_status=order_status, order_limit=order_limit
+        )
+        get_by_status_response = make_response(order_schemas.dump(orders_by_status))
+        return get_by_status_response
+
+    @order_blueprint.route("/orders/order_ingredients/<order_id>", methods=["GET"])
```
Comment (reynoldmorel) [inline src/flask/routes/order.py:65]: Rename to `/orders/<order_id>/ingredients`

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: src/flask/routes/order.py (hunk 2 above)
Comment (eapg) [reply src/flask/routes/order.py:65]: done.

File: src/flask/routes/order.py (hunk 1 above)
Comment (eapg) [reply src/flask/routes/order.py:56]: done

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**